src/main/java/org/jenkinsci/plugins/vsphere/builders/Clone.java

```diff
@@ -66,6 +69,12 @@ public Clone(String sourceName, String clone, boolean linkedClone,
         this.folder=folder;
         this.customizationSpec=customizationSpec;
         this.powerOn=powerOn;
+        if (timeoutInSeconds != null) {
+        	this.timeoutInSeconds = timeoutInSeconds;
```

Comment:
Once the timeoutInSeconds is an Integer, you should use ... = timeoutInSeconds.intValue(); to explicitly un-box the value.